Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue8228.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 8228, "[Bug]Grouped CollectionView shows the item source object name in each group", PlatformAffected.iOS)]
+	public class Issue8228 : TestContentPage
```

Comment:
This file is not added to the `Xamarin.Forms.Controls.Issues.Shared.shproj` and doesn't show up in the gallery app

Reply:
Also, would it be possible to add a UITest? Just a simple one that navigates here and checks that there is no ToString value of sorts

Reply:
Sorry for the late response!!
Included the file in shproj and added test to test to see if empty ones doesnt show up